### PR TITLE
V2

### PR DESCRIPTION
**Pull request by alanzeng17**
Getting rid of v2

### CODE REVIEW

**Comment by alanzeng17** [comment]
pls do it

**Comment by cSchubes** [comment]
looks awful lets do it

File: src/routes/colorRouter.js [hunk 1]
```diff
@@ -48,10 +48,15 @@ router.get('/', async (req, res, next) => {
...
-  colorVar.message = "Successfully added color to database."
-  res.status(HTTP_CODES.Ok).json(colorVar);
+  try {
+    let colorVar = req.body.color;
+    await knex.insert(colorVar).into('Colors');
+    colorVar.message = "Successfully added color to database."
+    res.status(HTTP_CODES.Ok).json(colorVar);
+  } catch (e) {
+    console.log(e);
+    res.status(400).json();
```
Comment (cSchubes) [inline src/routes/colorRouter.js]: can we return a message here? that way its consistent with how we respond in good conditions

**Review by cSchubes — APPROVED** [review]
Looks solid

**Pull request merged**